charts/owncloud/templates/deployment.yaml

```diff
@@ -48,61 +48,364 @@ spec:
+            - name: OWNCLOUD_LOG_DATE_FORMAT
+              value: {{ .Values.owncloud.log.dateFormat | quote }}
+            - name: OWNCLOUD_LOG_FILE
+              value: {{ .Values.owncloud.log.file | quote }}
+            - name: OWNCLOUD_LOG_LEVEL
+              value: {{ .Values.owncloud.log.level | quote }}
+            - name: OWNCLOUD_LOG_ROTATE_SIZE
+              value: {{ .Values.owncloud.log.rotateSize | quote }}
+            - name: OWNCLOUD_LOG_TIMEZONE
+              value: {{ .Values.owncloud.log.timezone | quote }}
+            - name: OWNCLOUD_LOST_PASSWORD_LINK
+              value: {{ .Values.owncloud.lostPasswordLink | quote }}
+            - name: OWNCLOUD_MAIL_DOMAIN
+              value: {{ .Values.owncloud.mail.domain | quote }}
+            - name: OWNCLOUD_MAIL_FROM_ADDRESS
+              value: {{ .Values.owncloud.mail.fromAddress | quote }}
+            - name: OWNCLOUD_MAIL_SMTP_AUTH
+              value: {{ .Values.owncloud.mail.smtp.auth | quote }}
+            - name: OWNCLOUD_MAIL_SMTP_AUTH_TYPE
+              value: {{ .Values.owncloud.mail.smtp.authType | quote }}
+            - name: OWNCLOUD_MAIL_SMTP_DEBUG
+              value: {{ .Values.owncloud.mail.smtp.debug | quote }}
+            - name: OWNCLOUD_MAIL_SMTP_HOST
+              value: {{ .Values.owncloud.mail.smtp.host | quote }}
+            - name: OWNCLOUD_MAIL_SMTP_MODE
+              value: {{ .Values.owncloud.mail.smtp.mode | quote }}
+            - name: OWNCLOUD_MAIL_SMTP_NAME
+              value: {{ .Values.owncloud.mail.smtp.name | quote }}
+            - name: OWNCLOUD_MAIL_SMTP_PASSWORD
+              value: {{ .Values.owncloud.mail.smtp.password | quote }}
+            - name: OWNCLOUD_MAIL_SMTP_PORT
+              value: {{ .Values.owncloud.mail.smtp.port | quote }}
+            - name: OWNCLOUD_MAIL_SMTP_SECURE
+              value: {{ .Values.owncloud.mail.smtp.secure | quote }}
+            - name: OWNCLOUD_MAIL_SMTP_TIMEOUT
+              value: {{ .Values.owncloud.mail.smtp.timeout | quote }}
+            - name: OWNCLOUD_MAINTENANCE
+              value: {{ .Values.owncloud.maintenance | quote }}
+            - name: OWNCLOUD_MARKETPLACE_CA
+              value: {{ .Values.owncloud.marketplace.ca | quote }}
+            - name: OWNCLOUD_MARKETPLACE_KEY
+              value: {{ .Values.owncloud.marketplace.key | quote }}
+            - name: OWNCLOUD_MAX_EXECUTION_TIME
+              value: {{ .Values.owncloud.maxExecutionTime | quote }}
+            - name: OWNCLOUD_MAX_FILESIZE_ANIMATED_GIFS_PUBLIC_SHARING
+              value: {{ .Values.owncloud.maxFilesizeAnimatedGifsPublicSharing | quote }}
+            - name: OWNCLOUD_MAX_INPUT_TIME
+              value: {{ .Values.owncloud.maxInputTime | quote }}
+            - name: OWNCLOUD_MAX_UPLOAD
+              value: {{ .Values.owncloud.maxUpload | quote }}
+            - name: OWNCLOUD_MEMCACHED_ENABLED
+              value: {{ .Values.owncloud.memcachedEnabled | quote }}
```

Comment:
Memcached options namespacing?  `.Values.owncloud.memcached.enabled`

Reply:
Yes that's exactly why this is on the TODO list above. I'm not sure if we should split it, as some settings start with `memcached` and some with `memcache` without `d`. If we switch to name spacing we lose the direct transferability from the docker name.